### DISCUSSION TITLE
[A-L] (2023/24) Foglio 13 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#278] [Esercizi - Viaggi/Piazza]
**Esercizio 5.8 del libro di testo (Geometria Analitica: Marco Abate & Chiara De Fabritiis)**

- Duplicato #277

**Comment by daromacarlo** [comment]
![Immagine 2023-12-04 175833](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/2cf763d2-5fca-48bf-9d0f-1251e93ce3cd)

**Comment by daromacarlo** [comment]
ho come un Déjà vu

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/c82ef14d-d6aa-455b-b83a-60c11d61b6a2)
